library/src/main/java/org/apache/apex/malhar/lib/state/spillable/inmem/InMemSpillableStateStore.java

```diff
@@ -75,6 +84,15 @@ public void put(long bucketId, @NotNull Slice key, @NotNull Slice value)
       store.put(bucketId, bucket);
     }
 
+    if (key.getClass() == Slice.class) {
```

Comment:
This code repeats in multiple places. Until we replace Slice, please keep the code in SliceUtil or another central place.